pkg/models/model.go

```diff
@@ -136,7 +136,7 @@ func (m ManagedModel) IsDeletedAt(t time.Time) bool {
 }
 
 type NamespacedModel struct {
-	Namespace string `json:"-" yaml:"-"`
```

Comment:
This is an ancient heritage from the old times where this was used in the API layer. This makes event sending quite hard, so let's get rid of this now that API layer does not depend on this anymore.